yarn-project/noir-protocol-circuits-types/src/noir_test_gen.test.ts

```diff
@@ -50,6 +52,8 @@ describe('Data generation for noir tests', () => {
   const format = (obj: object) => JSON.stringify(obj, null, 2).replaceAll('"', '');
 
   test.each(contracts)('Computes contract info for %s', async contract => {
+    const defaultVkHash = await hashVK(new Array(CLIENT_IVC_VERIFICATION_KEY_LENGTH_IN_FIELDS).fill(Fr.ZERO));
+    contract.privateFunctions.forEach(p => (p.vkHash = defaultVkHash));
```

Comment:
I edited this because there was a mismatch in the noir fixtures - the nr versioned hashed the vk, while the ts did not. This never became an issue because of the above comment.